python-sdk/tests_integration/databricks_tests/test_load.py

```diff
@@ -84,6 +84,37 @@ def test_delta_load_file_gcs(database_table_fixture):
     database.drop_table(table)
 
 
+@pytest.mark.integration
+@pytest.mark.parametrize(
+    "remote_files_fixture",
+    [
+        {"provider": "azure", "file_create": False},
+    ],
+    indirect=True,
+    ids=["azure_blob_storage"],
+)
+@pytest.mark.integration
+@pytest.mark.parametrize(
+    "database_table_fixture",
+    [
+        {
+            "database": Database.DELTA,
+        }
+    ],
+    indirect=True,
+    ids=["delta"],
+)
+def test_delta_load_file_azure_wasb(database_table_fixture, remote_files_fixture):
+    file_ = File(path=remote_files_fixture[0], conn_id="wasb_conn_with_sas_token")
+    database, table = database_table_fixture
+    database.load_file_to_table(
```

Comment:
At the moment, this test fails at `src/astro/databricks/load_file/load_file_job.py:88: in load_file_to_delta
    create_and_run_job` with the exception:
```
 E           airflow.exceptions.AirflowException: Databricks job failed. Job info ***'job_id': 764637050444463, 'run_id': 6507882, 'creator_user_name': 'daniel@astronomer.io', 'number_in_job': 6507882, 'state': ***'life_cycle_state': 'TERMINATED', 'result_state': 'FAILED', 'state_message': '', 'user_cancelled_or_timedout': False***, 'task': ***'spark_python_task': ***'python_file': 'dbfs:/mnt/pyscripts/load_file__tmp_fd003tmarfyucghqvccomivabj0fvuyzwq99b9a55bd01nndamj4p38gf.py'***, 'cluster_spec': ***'existing_cluster_id': '***'***, 'cluster_instance': ***'cluster_id': '***', 'spark_context_id': '7878723590138424515'***, 'start_time': 1673345570903, 'setup_duration': 1000, 'execution_duration': 667000, 'cleanup_duration': 0, 'end_time': 1673346238960, 'run_name': 'Untitled', 'run_page_url': 'https://dbc-9c390870-65ef.cloud.databricks.com/?o=4256138892007661#job/764637050444463/run/6507882', 'run_type': 'SUBMIT_RUN', 'attempt_number': 0, 'format': 'SINGLE_TASK'***
```
https://github.com/astronomer/astro-sdk/actions/runs/3882266043/jobs/6622222255